cmd/kube-scheduler/app/server.go

```diff
@@ -172,7 +172,7 @@ func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}, regis
 	// Create the scheduler.
 	sched, err := scheduler.New(cc.Client,
 		cc.InformerFactory.Core().V1().Nodes(),
-		cc.PodInformer,
+		cc.InformerFactory.Core().V1().Pods(),
```

Comment:
It doesn't look right. Original PodInformer is customized to filter out Succeeded/Failed pods:

https://github.com/kubernetes/kubernetes/blob/46e58df837ad83febd3fa1330c07bc0d353d406f/pkg/scheduler/factory/factory.go#L539-L548

Reply:
Oh, you are right. I'll fix it.